common/v2/features/index.ts

```diff
@@ -28,3 +28,4 @@ export { SignAndVerifyMessage } from './SignAndVerifyMessage';
 export { NotificationsProvider } from './NotificationsPanel';
 export { BroadcastTransactionFlow } from './BroadcastTransaction';
 export { SwapAssetsFlow } from './SwapAssets';
+export { ToastProvider } from './Toasts';
```

Comment:
So this is tricky. Toasts can be triggered by any part of the app and by placing it in features we run the risk of increasing circular dependencies.
I'm open to opinions, but believe it would be better to move it into either services or components.

Reply:
Yeah, I thought about this as well. But shouldn't we be moving Notifications as well then? Maybe moving both of them to service?